modules/database-commons/src/main/java/org/testcontainers/ext/ScriptUtils.java

```diff
@@ -278,19 +278,33 @@ public static void executeDatabaseScript(DatabaseDelegate databaseDelegate, Stri
 		}
 	}
 
+	public static class ScriptLoadException extends RuntimeException {
+		public ScriptLoadException(String message) {
+			super(message);
+		}
+
+		public ScriptLoadException(String message, Throwable cause) {
+			super(message, cause);
+		}
+	}
+
     private static class ScriptParseException extends RuntimeException {
         public ScriptParseException(String format, String scriptPath) {
             super(String.format(format, scriptPath));
         }
     }
 
     public static class ScriptStatementFailedException extends RuntimeException {
+		public ScriptStatementFailedException(String statement, int lineNumber, String scriptPath) {
```

Comment:
indentation error. Also, could you please call `this(statement, lineNumber, scriptPath, null)` and then call different supers if ex is null or not, so that `String.format` code will be shared (Didn't you say you hate code duplication? ;) )

Reply:
>indentation error.

Intellij Idea shows that's everything is ok :smiley_cat: 
![screenshot from 2018-02-08 16-12-21](https://user-images.githubusercontent.com/5352029/35974635-ebf389ec-0cea-11e8-9f58-b8a70eee7452.png)
> Also, could you please call this(statement, lineNumber, scriptPath, null) and then call different supers if ex is null or not, so that String.format code will be shared

Good idea, done, thanks!
> (Didn't you say you hate code duplication? ;) )

I did, but I don't like passing nulls into not mine methods :) But currently no NPE here, so I've done it.